tox.ini

```diff
@@ -26,38 +26,29 @@
 
 [tox]
 envlist =
-  py26,py27,py32,py33,py34,
-  py27-monolithic,py33-monolithic,py34-monolithic,
-  py27-not-relaxed-strides,py33-not-relaxed-strides,py34-not-relaxed-strides
+  py27,py34,py35,py36,
+  py27-not-relaxed-strides,py34-not-relaxed-strides,py35-not-relaxed-strides,py36-not-relaxed-strides
```

Comment:
Why drop the monolithic (whatever they are) builds? Not sure we need all the `not-relaxed-strides` testing either, as `relaxed-strides` is now the default,  but in any case that line should be broken.

@njsmith @pv Thoughts?